code/src/org/txstate/auto_scheduler/CourseInfo.java

```diff
@@ -0,0 +1,74 @@
+package org.txstate.auto_scheduler;
+
+/**
+ * Represents a course for the school
+ * @author Barry Rominger
+ */
+
+public class CourseInfo {
+    private String course_number;
+    private String course_name;
+    private String subject;
+
+    /**
+     * Gets the course number
+     * @return this course number
+     */
+
+    public String getCourse_number() {
+        return course_number;
+    }
+
+    /**
+     * Changes the course number
+     * @param course_number
+     */
+
+    public void setCourse_number(String course_number) {
+        this.course_number = course_number;
+    }
+
+    /**
+     * Gets the course name
+     * @return this course name
+     */
+
+    public String getCourse_name() {
+        return course_name;
+    }
+
+    /**
+     * Changes course name
+     * @param course_name
+     */
+    public void setCourse_name(String course_name) {

```

Comment:
please use camelized Names for method names like
`setCourseName`
`getCourseName`